File: Assets/Scripts/Carta.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.UI;
using Newtonsoft.Json;

public class Carta 
{

    public string nombreImagen;
    public int ataque;
    public bool esHechizo;
    public int id;
    public bool efecto;
    public int idJugador;

    public bool cartaJugada;


    // Start is called before the first frame update
    public Carta(string _nombreImagen, int _ataque, int _id, bool _efecto, bool _esHechizo){
        nombreImagen = _nombreImagen;
        ataque = _ataque;
        id = _id;
        esHechizo = _esHechizo;
        efecto = _efecto;
    }

    public void SetJugador(int id){
        idJugador = id;
    }
}

File: Assets/Scripts/rlcard-master/rlcard/games/HDL/Carta.py

class HDLCarta:
    def __init__(self, nombreImagen, ataque, id, efecto, hechizo):
        self.nombreImagen = nombreImagen
        self.ataque = ataque
        self.id = id
        self.efecto = efecto
        self.hechizo = hechizo

    def SetJugador(self, id):
        self.id = id  